homeassistant/helpers/trigger.py

```diff
@@ -78,7 +78,9 @@ class PluggableActionsEntry:
     """Holder to keep track of all plugs and actions for a given trigger."""
 
     plugs: set[PluggableAction] = field(default_factory=set)
-    actions: dict[object, tuple[HassJob, dict[str, Any]]] = field(default_factory=dict)
+    actions: dict[
+        object, tuple[HassJob[..., Coroutine[Any, Any, None]], dict[str, Any]]
```

Comment:
Should it not then be:
`HassJob[[dict[str, Any], Context | None], Coroutine[Any, Any, None]]`
rather than
`HassJob[..., Coroutine[Any, Any, None]`

Reply:
Pushed a new commit as it should indeed work in this case.
One potential issue could have been that `context` has a default parameter. It isn't possible to express those with the current typing syntax when specializing a generic TypeVar / ParamSpec.